modular_skyrat/modules/modular_vending/code/clothesmate.dm

```diff
@@ -84,6 +84,13 @@
 		/obj/item/clothing/head/fedora/white = 5,
 		/obj/item/clothing/head/fedora/fedblack = 5,
 		/obj/item/clothing/head/fedora/fedbrown = 5,
+		/obj/item/clothing/suit/jacket/leather/polychromic = 5,
+		/obj/item/clothing/under/misc/polyshorts = 5,
+		/obj/item/clothing/under/misc/polyjumpsuit = 5,
```

Comment:
already in the clothesmate

Reply:
I dont know where you're seeing them in the clothesmate but I cant find them being in there at all. I might be blind though *shrug